packages/components/src/Modal/Modal.module.css

```diff
@@ -109,7 +109,7 @@
 /* This is in a correct position and order */
 /* stylelint-disable-next-line no-descending-specificity */
 .rightAction > * {
-  margin-left: var(--space-small);
+  margin-left: var(--space-small) !important;
```

Comment:
@jdeichert I'm glad we landed a fix, but is there a version of this where we actually rebuild the modal actions using our own layout components to avoid needing custom CSS in the Modal?